src/main/java/com/o19s/es/ltr/action/ListStoresAction.java

```diff
@@ -38,7 +38,7 @@
 
 public class ListStoresAction extends Action<ListStoresAction.ListStoresActionRequest,
         ListStoresAction.ListStoresActionResponse,ListStoresAction.ListStoresActionRequestBuilder> {
-    public static final String NAME = "ltr:featurestore/list";
+    public static final String NAME = "cluster:admin/ltr/featurestore/list";
```

Comment:
should this be cluster:admin? would this not be too much open privileges? what do you think?

Reply:
This would almost be best as something like `indices:data/read/ltr/featurestore/list`, as all this really does is read from the index. I'll have to poke around to see if we can actually use that though.

Reply:
I like that one, I would definitely revisit the names to make them less general, specially because xpack (to the top of my head) uses this names to decide of authorisation.